components/button/button.tsx

```diff
@@ -151,7 +156,7 @@ const InternalCompoundedButton = React.forwardRef<
 
   useEffect(() => {
     // FIXME: for HOC usage like <FormatMessage />
-    if (!buttonRef || !(buttonRef as any).current || autoInsertSpaceInButton === false) {
+    if (!buttonRef || !(buttonRef as any).current || mergedAutoInsertSpace === false) {
```

Comment:
```suggestion
    if (!buttonRef || !(buttonRef as any).current || mergedAutoInsertSpace) {
```
经过上面的转换这里已经是个 boolean 了